<launch>  
       <!-- Mitsubishi_melfa Node-->  
       <node pkg="kinect" type="objects_detection" name="objects_detection" output="screen" launch-prefix="xterm -e" />  
       <node pkg="kinect" type="objects_trajectory" name="objects_trajectory" output="screen"/>         
</launch>



